src/main.ts

```diff
@@ -137,7 +137,7 @@ var __component__ = /*#__PURE__*/__normalizer(
 
   let resolvedMap: RawSourceMap | undefined = scriptMap
 
-  output.push(`export default __component__.exports`)
+  output.push(`export default /*#__PURE__*/ __getExports(__component__)`)
```

Comment:
We could also have not exported the `getExports` function from the normalizer module and instead just used an IIFE:
```ts
output.push(`export default /*#__PURE__*/ (function() { return __component__.exports })()`)
```

But I thought the exported function was more clear.